### PR TITLE
Fixes #30553 - Add foreman_statistics plugin

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: config/katello.migrations/200803163936-add-statistics-plugin.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+answers['foreman::plugin::statistics'] ||= false
```
Comment (tbrisker) [inline config/katello.migrations/200803163936-add-statistics-plugin.rb:1]: Not sure how much it matters, but you're missing a `20` in the file name

Comment (ekohl) [reply config/katello.migrations/200803163936-add-statistics-plugin.rb:1]: It was added by the script, in katello they all skip the leading `20`.

**Comment by ekohl** [comment]
Tests are failing because I didn't update the answers in the migration tests.

File: config/katello.migrations/200803163936-add-statistics-plugin.rb (hunk 1 above)
Comment (tbrisker) [reply config/katello.migrations/200803163936-add-statistics-plugin.rb:1]: ok, then we should be good until 2100 ;)

**Review by ehelms — APPROVED** [review]

**Pull request merged**